### DISCUSSION TITLE
AMapImageryProvider，VcImageryProviderBaidu初始化时无法自动加载

### DISCUSSION THREAD

**Discussion by kfcvs** [zouyaoji/vue-cesium#195] [Q&A] [answered]
百度地图
![image](https://user-images.githubusercontent.com/31976998/164874487-af6a789b-0f29-4ce6-8bef-877fceb5ef42.png)
高德地图
![image](https://user-images.githubusercontent.com/31976998/164874506-db90cab2-a74a-455b-9d11-9dd97d37db84.png)

**Comment by zouyaoji** [comment, marked as answer]
我这看没问题呢。

**Reply by kfcvs** [reply]
地球缩小时不显示，放大到一定级别后才能显示

**Reply by zouyaoji** [reply]
明白了，我目前也不清楚这是不是bug呢，你看能不能考虑控制scene缩放，不要球缩放到太小呢。
我看其他平台加载的似乎也有这个问题。
